lib/Common/Memory/HeapAllocator.cpp

```diff
@@ -303,10 +339,27 @@ void NoThrowNoMemProtectHeapAllocator::ClearTrackAllocInfo(TrackAllocData* data
 #endif // TRACK_ALLOC
 #endif
 
-#if defined(HEAP_TRACK_ALLOC) || defined(INTERNAL_MEM_PROTECT_HEAP_ALLOC)
+HeapAllocator::HeapAllocator(bool useAllocMemProtect)
+    : m_privateHeap(nullptr)
```

Comment:
@dreamer-dead I now initialize m_privateHeap to nullptr correctly in the constructor.